### PR TITLE
SWI-3665 Update SDK Based on Recent Spec Changes

### PR DESCRIPTION
**Pull request by DX-Bandwidth**
Auto-generated by Update SDK Workflow

### CODE REVIEW

**Review by brianluisgomez — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Comment by ajrice6713** [comment]
Putting this in draft - we cant add tests till GET /calls goes GA on 8/9

**Review by ckoegel — APPROVED** [review]

File: src/test/java/com/bandwidth/sdk/api/CallsApiTest.java [hunk 1]
```diff
@@ -61,7 +62,7 @@ public class CallsApiTest {
     private static CallbackMethodEnum callbackMethod = CallbackMethodEnum.POST;
     private static String testCallId = "Call-Id";
     private static String testXmlBody = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><Bxml><SpeakSentence locale=\"en_US\" gender=\"female\" voice=\"susan\">This is a test bxml response</SpeakSentence><Pause duration=\"3\"/></Bxml>";
-    private static int TEST_SLEEP = 3;
+    private static int TEST_SLEEP = 6;
```
Comment (ajrice6713) [inline src/test/java/com/bandwidth/sdk/api/CallsApiTest.java:65]: I think we want to set this to 40 to mimic voice api teams alarm, right @ckoegel ?

File: src/test/java/com/bandwidth/sdk/api/ConferencesApiTest.java [hunk 2]
```diff
@@ -62,8 +62,8 @@ public class ConferencesApiTest {
     private static URI answerUrl;
     private static URI conferenceRedirectUrl;
     private static String updateRecordingBxml = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><Bxml><StartRecording/><SpeakSentence locale=\"en_US\" gender=\"female\" voice=\"susan\">This should be a conference recording.</SpeakSentence><StopRecording/></Bxml>";
-    private static int TEST_SLEEP = 3;
-    private static int MAX_RETRIES = 40;
+    private static int TEST_SLEEP = 6;
```
Comment (ajrice6713) [inline src/test/java/com/bandwidth/sdk/api/ConferencesApiTest.java:65]: Same here, set the sleep to 40 and dont retry.

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: src/test/java/com/bandwidth/sdk/api/ConferencesApiTest.java (hunk 2 above)
Comment (ckoegel) [reply src/test/java/com/bandwidth/sdk/api/ConferencesApiTest.java:65]: Here I believe this is used for waiting for the recording to finish, so we want to sleep less and retry to poll manteca

File: src/test/java/com/bandwidth/sdk/api/CallsApiTest.java (hunk 1 above)
Comment (ckoegel) [reply src/test/java/com/bandwidth/sdk/api/CallsApiTest.java:65]: this one should also be fine cause we use it in multiple spots, we should just update the sleep for get call state to be 40s

File: src/test/java/com/bandwidth/sdk/api/CallsApiTest.java [hunk 3]
```diff
@@ -194,20 +195,41 @@ public void createCallForbidden() throws ApiException {
...
+    public void getCallState() throws ApiException, InterruptedException {
+        Basic.setUsername(BW_USERNAME);
+        Basic.setPassword(BW_PASSWORD);
 
-    //     assertThat(response.getStatusCode(), anyOf(is(200),is(404)));
-    //     assertThat(response.getData(), hasProperty("callId", is(instanceOf(String.class))));
-    //     assertThat(response.getData(), hasProperty("state", is(instanceOf(String.class))));
-    //     assertThat(response.getData(), hasProperty("direction", is(CallDirectionEnum.OUTBOUND)));
-    // }
+        TimeUnit.SECONDS.sleep(TEST_SLEEP);
```
Comment (ckoegel) [inline src/test/java/com/bandwidth/sdk/api/CallsApiTest.java:225]:
```suggestion
        TimeUnit.SECONDS.sleep(40);
```

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**